ros_ws/src/dataset_generation/src/filter_point_clouds_node.cpp

```diff
@@ -8,7 +8,7 @@ int main()
         "/home/parshwa/Desktop/CMPE_255 Project/bag5_2020-05-05-15-11-17.bag-20200506T192755Z-001/bag5_2020-05-05-15-11-17.bag/Jeep.pcd";
     std::string out_folder_pcd =
         "/home/parshwa/Desktop/CMPE_255 Project/bag5_2020-05-05-15-11-17.bag-20200506T192755Z-001/bag5_2020-05-05-15-11-17.bag/Jeep_voxelized,pcd";    
```

Comment:
I think you have a typo at the end of the line before `pcd`. It should be a `.`, but looks like a `,`